src/website/views/components/date-picker/_examples.pug

```diff
@@ -182,6 +182,44 @@ p.-text
       <chi-date-picker id="date-excluded-days-dates" excluded-weekdays="0, 6" excluded-dates="01/24/2020, 05/25/2020, 07/03/2020, 09/12/2020, 12/12/2020"></chi-date-picker>
   #html-excluded-days-dates.chi-tabs-panel
 
+h3 Error
+p.-text
+  | Use the <code>danger</code> state to provide feedback to users when date fails to validate.
+  | To meet accessibility requirements, danger date picker must include an error message explaining the
```

Comment:
If the message is present, we will need to bring back the danger message HTML you had earlier but providing it in the code snippet as well.
Example:
```
<div class="chi-label -status -danger">
  <chi-icon icon="circle-warning"></chi-icon>
  Please select a date.
</div>
```